pkg/helm/handlers/handlers.go

```diff
@@ -16,10 +18,29 @@ var (
 	plog = capnslog.NewPackageLogger("github.com/openshift/console", "helm")
 )
 
+func NewHelmHandler(apiUrl string, transport http.RoundTripper) *HelmHandlers {
+	return &HelmHandlers{
```

Comment:
let's give a better name to this structure, it is a kind of context, thus let's call it `helmContext` - the structure does not need to public anymore.